src/runtime/basic-runtime.js

```diff
@@ -507,6 +507,13 @@ export class ConfiguredBasicRuntime {
           lastContributionsFlow.showNoEntitlementFoundToast();
           return;
         }
+
+        const lastAudienceActionFlow =
+          this.autoPromptManager_.getLastAudienceActionFlow();
+        if (lastAudienceActionFlow) {
+          lastAudienceActionFlow.showNoEntitlementFoundToast();
```

Comment:
It feels strange to push this into all iframes. There isn't a singular one we could re-use to do that?

Reply:
Since subscription / contribution / regwall and newsletter iframes share the same feature, I think it's reasonable to add this piece here.

Reply:
Sounds good